saleor/tax/utils.py

```diff
@@ -248,3 +249,24 @@ def get_shipping_tax_class_kwargs_for_order(tax_class: Optional["TaxClass"]):
         "shipping_tax_class_private_metadata": tax_class.private_metadata,
         "shipping_tax_class_metadata": tax_class.metadata,
     }
+
+
+def check_negative_values_in_tax_data(tax_data: TaxData) -> bool:
```

Comment:
Maybe we should also return info on which line is negative?

Reply:
The details will be visible in logs attribute